src/Baballonia.IPCameraCapture/IPCameraCapture.cs

```diff
@@ -23,12 +22,13 @@ public sealed class IpCameraCapture(string url, ILogger logger) : Capture(url, l
 
     public override bool CanConnect(string connectionString)
     {
-        return Uri.TryCreate(connectionString, UriKind.RelativeOrAbsolute, out _);
+        return Uri.TryCreate(connectionString, UriKind.RelativeOrAbsolute, out _) && connectionString.StartsWith("http://");
     }
 
     public override Task<bool> StartCapture()
     {
-        Task.Run(() => StartStreaming(Source, null, null, _cancellationTokenSource.Token)); // Size of Babble frame
+        Task.Run(() => StartStreaming(Source, null, null, _cancellationTokenSource.Token, 1024,
+            256 * 256)); // Size of Babble frame
```

Comment:
We can drop the `256*256` here, this is the (max) size of an incoming frame, not the expected size.